OrbitGl/SamplingReportDataView.cpp

```diff
@@ -113,9 +113,29 @@ void SamplingReportDataView::DoSort() {
       break;
   }
 
-  if (sorter) {
-    std::stable_sort(indices_.begin(), indices_.end(), sorter);
+  if (!sorter) {
+    return;
   }
+
+  const auto fallback_sorter = [&](const auto& ind_left, const auto& ind_right) {
+    // address is the absolute function address. Hence it is unique and qualifies
```

Comment:
Also start with capital letters in Comments.

Reply:
Well, I used "address" because I meant the field name. I will add backticks to make that clear.

Reply:
Thanks.